# HG changeset patch
# User Rik <rik@octave.org>
# Date 1545073237 28800
#      Mon Dec 17 11:00:37 2018 -0800
# Node ID 741968251fca0b6f2be534999d3afef53866eb43
# Parent  339db43aa056187aec5807c69ae222cc62d83b7e
legend.m: Update failing BIST test now that "bestoutside" is implemented.

* legend.m: Remove BIST test looking for failure when "bestoutside" location is
used.

diff --git a/scripts/plot/appearance/legend.m b/scripts/plot/appearance/legend.m
--- a/scripts/plot/appearance/legend.m
+++ b/scripts/plot/appearance/legend.m
@@ -1821,12 +1821,11 @@ endfunction
 %! end_unwind_protect
 
 %!test
 %! ## Test warnings about unsupported features
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   plot (1:10);
 %!   fail ("legend ('location','best')", "warning", "'best' not yet implemented");
-%!   fail ("legend ('location','bestoutside')", "warning", "'best' not yet implemented");
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
